index.md

```diff
@@ -93,7 +93,7 @@ If you cannot find contacts on the website, other places to look include:
 Describe the accessibility barrier clearly. This will help the organization find and fix the problem. Include:
 
 *   the page where you found a problem
-*   what the problem is
+*   what the problem is that you observe
```

Comment:
Many may not know what the problem actually is, but they can state what they are perceiving.